java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/scope/CurrentJavaProjectDependenciesScopeProvider.java

```diff
@@ -98,7 +98,7 @@ public boolean initialize(Lookup context, AtomicBoolean cancel) {
         }
         scope = Scope.create(Arrays.asList(projectSources), null, null, true);
         detail = pi.getDisplayName();
-        icon = new ImageIcon(ImageUtilities.mergeImages(
+        icon = ImageUtilities.image2Icon(ImageUtilities.mergeImages(
                 ImageUtilities.icon2Image(pi.getIcon()),
                 ImageUtilities.loadImage("org/netbeans/modules/refactoring/java/resources/binary_badge.gif"),
                 10, 10));
```

Comment:
would this now also work?

```java
        icon = ImageUtilities.mergeIcons(
                pi.getIcon(),
                ImageUtilities.loadIcon("org/netbeans/modules/refactoring/java/resources/binary_badge.gif"),
                10, 10);
```

Reply:
Yes, missed that one! Will do the latter now.

Reply:
Added another commit that simplifies as few such cases, including that one.